flyMultiPlateScript.m

```diff
@@ -200,14 +208,42 @@ function flyMultiPlateScript()
     timestampFileName = strrep(timestampFileName,'.mp4','-timestamps.csv');
     timestampFileName = strrep(timestampFileName,'.m4v','-timestamps.csv');
 
-    %Check the existence of the associated timstamp file
     curFolder = pwd;
+
+    %Check the existence of the associated timstamp file
     cd(pathName);
     if not(exist(timestampFileName,'file') == 2)
         [timestampFileName,timestampPathName] = uigetfile({'*.csv'},strcat('Select the timestamp file associated with: ',fileName));
+        %Changing to this directory assuming we'll find the wellposes file there too
+        cd(timestampPathName);
     else
         timestampPathName = pathName;
     end
+
+    if useSavedWells == 1
+
+        %% Find the well positions file
+
+        % Assume the well positions file is in the same place as the timestamps file
+        wellposesFileName = strrep(timestampFileName,'-wellposes');
+        tmpwellposesFileName = strcat(wellposesFileName,'.mat');
+        wellposesPathName = timestampPathName;
+
+        %Check the existence of the associated wellposes file
+        if not(exist(tmpwellposesFileName,'file') == 2)
+            [tmpwellposesFileName,wellposesPathName] = uigetfile({'*.csv'},strcat('Select the timestamp file associated with: ',fileName));
+            %Remove the .mat for the load function (which doesn't take the filename with '.mat')
```

Comment:
OK. I wasn't sure whether it would take the full name or not.  The doc page I got it from didn't supply .mat and said that it opened a file with .mat:

https://www.mathworks.com/help/matlab/matlab_env/save-load-and-delete-workspace-variables.html